### ISSUE TITLE
Tempoary file littering root directory

### ISSUE THREAD

**Issue by TimPerry** [garris/BackstopJS#319]
Howdy,

It looks like [this line](https://github.com/garris/BackstopJS/blob/786078a5caa3008fedf4e664e6f85121345eec19/core/util/makeConfig.js#L70) is creating a temporary file in the root directory, is this expected? I find it kinda this odd that it doesn't default to the backstop_data folder. Not sure if this is something that I am meant to be checking into VC. Or on that matter exactly what I should be checking in.

I am using this in docker with the `6.5-onbuild` image, so npm `3.10.3` and node `6.5.0`. Ow and my package.json looks like this:

```
    "backstopjs": "^2.0.4",
```

This is an example file it creates `116914-532-1aesg60.gs95nlv7vi.json` that has this in it:

```
{
  "compareConfig": {
    "testPairs": []
  }
}
```

p.s. love this project :)

Cheers

Tim

**Comment by TimPerry** [comment]
Was just having a read through the packages readme on npm and it appears that they should be clearing that file up :/

**Comment by TimPerry** [comment]
Looks like there are a lot of open issues on that project. Might be a good idea to swap to an alternate package? If you are busy I might be able to get a PR sorted.

**Comment by garris** [comment]
@timperry PR's welcome.  Cheers.

**Comment by chipcullen** [comment]
I don't know if this will help matters or not, but I was running into this same issue.

1. I was running backstop locally, not globally
2. I was having to run backstop with `sudo`

Re: 2 - I was having to run in sudo because when I didn't, I got the following error:

```
 COMMAND | Error: EACCES: permission denied, open '/Users/user/project/node_modules/backstopjs/capture/config.json'
                    at Error (native)
```

I'm on a mac. When I tracked that particular file in the Finder, the system had read/write access, but not my user. I manually added myself to the file's permissions, and then suddenly I didn't need to run the commands with `sudo`. And after _that_ point,  I was no longer getting the random .json file.
